deploy.bat

```diff
@@ -1 +1,3 @@
-robocopy ".\front_end" "C:\inetpub\wwwroot\DataWeb\IbexDataweb" /S
\ No newline at end of file
+cd /d %~dp0
+robocopy /S ".\front_end" "C:\inetpub\wwwroot\DataWeb\IbexDataweb" /S
+pause
```

Comment:
are both /S required? And do we want to mirror i.e. if we remove a file from .\front_end do we wish that change to be propagated?

Reply:
hmm, no we don't need both S's and yes probably mirroring is good ....